_posts/pl/2022-01-11-smarthome.md

```diff
@@ -0,0 +1,316 @@
+
+![Taśma Yeelight](/assets/img/posts/2022-01-11-smarthome/yellightstrip.png){: width="250" }
+
+Zalety — ładnie świeci, możliwy wybór koloru, łatwo się przedłuża.
+
+Wady — maksymalna długość taśmy w niektórych przypadkach może okazać się za krótka. 
+Łączenie kolejnych „kawałków” jest zrobione dosyć dużym wtykiem, w porównaniu do taśm, które lutujemy i może spowodować problem z umieszczeniem całości w aluminiowym profilu na taśmy.
+
+#### Przekaźnik Shelly 2.5
+
+![Przekaźnik Shelly](/assets/img/posts/2022-01-11-smarthome/shelly25.png){: width="250" }
+
+Zalety — duża liczba dostarczanych danych, a także różne sposoby integracji (API lub MQTT).
+Każdy z dwóch kanałów może być osobno konfigurowany pod względem rodzaju podłączonego klawisza (dzwonkowy, stanowy).
+Dla każdego z kanałów mamy też osobny zestaw informacji o aktualnym poborze prądu.
+Możemy skonfigurować stan przekaźnika po powrocie zasilania (dla mnie to ważne, aby pod nieobecność nie okazało się, że całe mieszkanie nagle się świeci).
+No i ostatnie — mieści się w puszce, niestety część urządzeń nie przewiduje umieszczania w europejskich okrągłych puszkach.
+
+Wady — komunikacja Wi-Fi i wszystko, co z nią się wiąże.
+Z doświadczenia preferuje Zigbee, tym bardziej że urządzenia Zigbee na stałe podłączone do prądu z automatu przedłużają zasięg sieci.
+
+#### ESP32
+
+![Esp32](/assets/img/posts/2022-01-11-smarthome/esp32.png){: width="250" }
+
+Mały, tani, ale potężny układ. Na standardowym wyposażeniu używanej przeze mnie wersji „devkit v1” znajduje się Wi-Fi oraz Bluetooth.
+Dzięki popularności układu łatwo można znaleźć projekty obudowy do samodzielnego wydrukowania na drukarce 3D.
+W moim SmartHome pracują dwa rodzaje takich układów:
+- Pierwszy z nich ma na pokładzie [OpenMQTTGateway](https://docs.openmqttgateway.com/). Wykorzystuje go do odczytywania danych z czujników Bluetooth jak czujnik w kwiatku (o czym dalej), czy zegarek z wyświetlaczem eInk.
+- Drugi z nich ma na pokładzie [EspHome](https://esphome.io/) i podłączony odbiornik oraz nadajnik podczerwieni. Myślę, że podłączone elementy już dobrze reprezentują jego przeznaczenie. 
+  W dalszych planach mam też przetłumaczenie sygnałów podczerwieni z indukcji Electrolux Hob2Hood na okap firmy Globalo (sprzęt, jaki posiadam).
+  Wyzwanie stanowi brak w okapie opcji „ustaw wentylator na 3" (taki sygnał nada indukcja). Jedynym sposobem osiągnięcia tego stanu jest klikanie w +/-.
+
+
+### Komunikacja po Bluetooth
+
+To, co można powiedzieć o tym sposobie integracji, że na pewno jego wadą jest mały zasięg. 
+Wcześniej wspomniany układ Esp32 z OpenMQTTGateway może okazać się niezbędnym narzędziem celem zebrania danych z różnych zakątków domu.
+Ciekawostką może być to, że sensory Bluetooth Low Energy, póki są niepołączone z konkretnym urządzeniem, nadają informacje o swoim stanie w eter w taki sposób,
+że odczytać może je każdy. Stąd odbieram dane z zegarka Xiaomi sąsiadów. Z urządzeń Bluetooth mam następujące:
```

Comment:
Ciekawostką może być to, że sensory Bluetooth Low Energy, póki są niepołączone z konkretnym urządzeniem, nadają informacje o swoim stanie w taki sposób, że każdy może je odczytać. Stąd odbieram dane z zegarka Xiaomi sąsiadów...
Urządzenia Bluetooth z których korzystam: